docs/contributing/test-data-models-overview.md

```diff
@@ -0,0 +1,250 @@
+# Test data models overview
+
+A test data model is set of helper objects commercetools API consumers can use during writing their own tests in order to mock the data returned by our services.
+
+These helpers are flexible enough to represent different types of responses from different APIs and with different values.
+
+## Public API
+
+Each test data model has the same API which consists on:
+
+- A Rest builder
+- A Graphql builder
+- Typescript types
+
+A `Builder` is the object you would be interacting with as a consumer and it has only two functions:
+
+- `random`: exposes a fluent API which allows to build a customized version of the data model
+- `presets`: predefined versions of the data model
+
+This is how it looks like the entry point of a data model package:
+
+```ts
+import { RestModelBuilder, GraphqlModelBuilder } from './builders';
+import * as GeometryPresets from './presets';
+
+export * from './types';
+
+export const GeometryRest = {
+  random: RestModelBuilder,
+  presets: GeometryPresets.restPresets,
+};
+
+export const GeometryGraphql = {
+  random: GraphqlModelBuilder,
+  presets: GeometryPresets.graphqlPresets,
+};
+```
+
+### Examples
+
+#### Building default random data models
+
+```ts
+import {
+  RestModelBuilder,
+  GraphqlModelBuilder,
+  type TGeometryRest,
+  type TGeometryGraphql,
+} from '@commercetools-test-data/geometry';
+
+const restGeometry: TGeometryRest = RestModelBuilder.random().build();
+const graphqlGeometry: TGeometryGraphql = GraphqlModelBuilder.random().build();
+```
+
+#### Building customized data models
+
+```ts
+import {
+  RestModelBuilder,
+  GraphqlModelBuilder,
+  type TGeometryRest,
+  type TGeometryGraphql,
+} from '@commercetools-test-data/geometry';
+
+const restGeometry: TGeometryRest = RestModelBuilder.random()
+  .coordinates([12.45, 66.33])
+  .build();
+const graphqlGeometry: TGeometryGraphql = GraphqlModelBuilder.random()
+  .coordinates([12.45, 66.33])
+  .build();
+```
+
+#### Building predefined version data models
```

Comment:
```suggestion
#### Building predefined presets of data models
```

Reply:
The way I see it, "presets" is what we use in order to have predefined versions of the data models, but the main concept is about having a deterministic version of a model.